File: DataAccessLayer/Models/FuelCost.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataAccessLayer.Models
{
    public class FuelCost
    {
        public int Id { get; set; }
        public DateTime DateIssued { get; set; }
        public TimeSpan TimeIssued { get; set; }
        public string Position { get; set; }
        public float Amount { get; set; }
        public decimal Price { get; set; }
    }
}

File: pdf-report-module/src/hr/mucnjakf/models/FuelCost.java
package hr.mucnjakf.models;

import javax.persistence.Embeddable;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.math.BigDecimal;
import java.sql.Date;
import java.sql.Time;

@Embeddable
@Entity
@Table(name = "FuelCosts")
public class FuelCost {
    @Id
    private int id;
    private Date dateIssued;
    private Time timeIssued;
    private String position;
    private float amount;
    private BigDecimal price;

    public FuelCost() {
    }

    public int getId() {
        return id;
    }

    public Date getDateIssued() {
        return dateIssued;
    }

    public Time getTimeIssued() {
        return timeIssued;
    }

    public String getPosition() {
        return position;
    }

    public float getAmount() {
        return amount;
    }

    public BigDecimal getPrice() {
        return price;
    }

    public String printToConsole() {
        return "Fuel cost: " + position + " - $" + price;
    }

    @Override
    public String toString() {
        return "FuelCost{" +
                "id=" + id +
                ", dateIssued=" + dateIssued +
                ", timeIssued=" + timeIssued +
                ", position='" + position + '\'' +
                ", amount=" + amount +
                ", price=" + price +
                '}';
    }
}
